src/Library/demos/Toggle Button/code.rs

```diff
@@ -0,0 +1,25 @@
+use crate::workbench;
+use gtk::prelude::*;
+use std::collections::HashMap;
+
+pub fn main() {
+    let buttons: HashMap<&str, &str> = [
+        ("button_no_look", "Don't look"),
+        ("button_look", "Look"),
+        ("button_camera", "Camera"),
+        ("button_flashlight", "Flashlight"),
+        ("button_console", "Console"),
+    ]
+    .iter()
+    .cloned()
+    .collect();
```

Comment:
You can use `from` instead: https://doc.rust-lang.org/std/collections/hash_map/struct.HashMap.html#method.from